extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -535,6 +537,85 @@ def from_dict(self, model):
         self._c_lw = copy.deepcopy(model['_c_lw'])
         self._c_l = copy.deepcopy(model['_c_l'])
 
+    def _make_json_serializable(self, model):
+        """Makes the StringClassifier dict JSON serializable by
+        converting numpy data types to Python data types.
+
+        Args:
+            model: A dict representing a StringClassifier.
+
+        Returns:
+            model: A dict representing a StringClassifier
+        """
+
+        conversion_list_of_list_properties = [
+            '_b_dl',
+            '_c_dl',
+            '_c_lw'
+        ]
+
+        conversion_list_properties = [
+            '_c_l'
+        ]
+
+        conversion_properties = [
+            '_l_dp'
+        ]
+
+        for classifier_property in conversion_list_of_list_properties:
```

Comment:
@seanlip . We could, but I don't think it makes much difference. What really would make a difference is if we can find a way to include this script : https://github.com/cmry/cmry.github.io/blob/master/sources/serialize_sk.py . Or maybe just write write this module locally. I don't think its a lot. This would reduce everything to one liners.